src/account/Education.tsx

```diff
@@ -42,6 +42,8 @@ interface EducationStep {
   // If set to true, title is displayed at the top
   isTopTitle?: boolean
   text?: string
+  valueProp?: string
```

Comment:
nit: `prop` is too much linked to React component properties
```suggestion
  valueProposition?: string
```

Reply:
Fixed in 00620f7aa24e884eef465e40f4eeb61559b5ba95!